RELEASE.md

```diff
@@ -38,14 +38,15 @@ pip install -ve .
 ### Install release dependencies
 
 ```bash
-conda install -c conda-forge nodejs babel twine build
+conda install -c conda-forge nodejs babel
 npm install -g po2json
+pip install jupyter_releaser  # used for build dependencies (build, twine, tbump)
 ```
 
 ### Update the version
 
 ```bash
-vim notebook/_version.py  # update version
+tbump --only-patch <new_version> # set the new version
 vim pyproject.toml  # update version to match
```

Comment:
I see `tbump` also updating `pyproject.toml`, should this line be removed as well?

Reply:
Good catch, thanks!